packages/embark-ui/src/components/TextEditor.js

```diff
@@ -25,10 +27,46 @@ const initMonaco = (value, theme) => {
   });
 };
 
+const Tab = SortableElement(({file, onTabClick, onTabClose, theme}) => {
+  return (
+    <li key={file.name} className={classNames("tab", "p-2", "pl-3", "pr-3", "list-inline-item", "mr-0", "border-right", "border-bottom", 
+    {
+      'border-light': LIGHT_THEME === theme,
+      'border-dark': DARK_THEME === theme
+    },
+    {
+      'active': file.active
+    })}>
+      <a
+        href="#switch-tab"
```

Comment:
Very minimal, but this should be 
```suggestion
        href="#switch-file"
```

Reply:
Fixed, thank you!